tasks/curator.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Add Curator apt repo.
+  apt_repository:
+    repo: "deb http://packages.elastic.co/curator/5/debian stable main"
+    state: present
+
+- name: Install Curator.
+  apt:
+    name: elasticsearch-curator
+    state: latest
+    update_cache: yes
+
+- name: Create Curator configuration directory.
+  file:
+    path: /etc/curator
+    state: directory
+    mode: "0755"
+    owner: root
+    group: root
+
+- name: Copy Curator configuration.
+  file:
+    src: curator.yml
+    dest: "/etc/curator/curator.yml"
+    mode: "0644"
+    owner: root
+    group: root
+
+- name: Copy Curator script to delete old indices.
+  template:
+    src: delete_indices.yml.j2
```

Comment:
See comment for `/etc/curator/curator.yml` -- this should also be all ansible vars and feed it directly through the `copy` module w/ `content`. That will make it super super easy for anyone to over-ride fields with ansible vars